fetch_someip_libs.sh

```diff
@@ -0,0 +1,23 @@
+echo "Clonning COMMONAPI libs"
```

Comment:
You should add a shebang. We can't know if this script will run in bash, zsh, dash, fish or any other shell.

```suggestion
#!/usr/bin/bash

echo "Clonning COMMONAPI libs"
```